### PR TITLE
release-it- use bamper plugin, pyproject update poetry settings

### PR DESCRIPTION
**Pull request by oharan2**
- Verified VERSION file
- Updated project metadata and dependencies
- Updated .release-it file `change-log` and `plugins` sections
- Updated `poetry.lock` according new changes

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,21 +24,39 @@ authors = ["Meni Yakove", "Ruth Netser"]
...
+homepage = "https://github.com/RedHatQE/cloud-tools"
+documentation = "https://github.com/RedHatQE/cloud-tools/blob/main/README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+# Ipython requires Python >=3.9
```
Comment (myakove) [inline pyproject.toml:36]: Please remove this comment, and set the project python to ^3.8

File: pyproject.toml [hunk 2]
```diff
@@ -24,21 +24,39 @@ authors = ["Meni Yakove", "Ruth Netser"]
...
+ipython = "^8.14.0"
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
+build-backend = "poetry_dynamic_versioning.backend"
+
+[tool.poetry-dynamic-versioning.substitution]
+files = ["VERSION"]
```
Comment (myakove) [inline pyproject.toml:58]: Need to be created and set the version there.

File: pyproject.toml [hunk 3]
```diff
@@ -24,21 +24,39 @@ authors = ["Meni Yakove", "Ruth Netser"]
 readme = "README.md"
```
Comment (myakove) [inline pyproject.toml:24]: `authors = ["Meni Yakove", "Ruth Netser"]` missing email, please add

**Review by myakove — CHANGES_REQUESTED** [review]
Update the PR title and add more info

**Review by rnetser — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**